Hymns/Display/SplashScreenView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+import Lottie
+
+//Implemented using https://www.raywenderlich.com/4503153-how-to-create-a-splash-screen-with-swiftui
+//And the help from https://www.youtube.com/watch?v=fVehE3Jf7K0
+struct SplashScreenView: UIViewRepresentable {
+    let animationView = AnimationView()
+    var filename = "firstLaunchAnimation"
```

Comment:
Is this ever edited? If not, can this be a let

Reply:
Changed in new PR